include/error.h

```
@@ -30,6 +30,7 @@
 #define KP_EENCRYPT         8
 #define KP_INVALID_MSG      9
 #define KP_EXIT             10
+#define KP_ENOENT           11
```

If you want to report an error with an existing code you should just set
```
errno = ENOENT
ret = KP_ERRNO
```